package netcore;
option cc_generic_services = true;
option java_generic_services = true;
option py_generic_services = true;

message TestRequest{
	required string testmsg = 1;
}

message TestResponse{
}

service TestService{
	rpc TestRpc(TestRequest) returns (TestResponse);
}